libs/transaction-handler/src/types/dao.ts

```diff
@@ -49,6 +49,7 @@ export function castCreateDao({
     id: daoId,
     config: args.config,
     ...castDaoPolicy({ daoId, daoPolicy: args.policy }),
+    metadata: Buffer.from(args.config.metadata, 'base64').toString('utf-8'),
```

Comment:
3 copy-pastes found. Can we move to utils though?

Reply:
Moved to utils